src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -119,6 +119,13 @@
     @Nullable
     OffsetDateTime getTimeBoosted();
 
+    /**
+     * Returns whether a member is boosting the guild or not.
+     *
+     * @return true if it is boosting, false otherwise.
```

Comment:
Not sure if that breaks consistency or not but I would prefer `{@code true|false}` over true|false.

Reply:
It wouldn't be consistent, but I found it should be True and not true, so I'll change it